src/test/ui-fulldeps/gated-plugin.rs

```diff
@@ -1,6 +1,8 @@
+// ignore-tidy-linelength
```

Comment:
We can probably just not list the whole error in this file? Truncating it should be fine since this is a UI test anyway.

Reply:
🤷‍♂ -- these files will be removed anyways so does it really matter? :P